apps/admin-portal/src/constants/application-constants.ts

```diff
@@ -56,6 +56,14 @@ export const APPLICATIONS_PAGE_PATH: string = "/applications";
  */
 export const USER_PORTAL_IDENTIFIER: string = "This is the user portal application.";
 
+/**
+ * Error description when the user denies consent to the app
+ * @constant
+ * @type {string}
+ * @default
+ */
+export const USER_DENIED_CONSENT: string = "User denied the consent";
```

Comment:
Since the variable is already assigned with a string, the type string is redundant. Could we remove it?

Reply:
Done